simulation_ws/src/cloudwatch_simulation/launch/small_house_turtlebot_navigation.launch.py

```diff
@@ -10,15 +10,15 @@ def generate_launch_description():
     ld = launch.LaunchDescription([
         launch.actions.DeclareLaunchArgument(
             name='x_pos',
-            default_value='3.3'
+            default_value='3.5'
```

Comment:
Does the (0, 0) in small house world not being spawnable still need to be addressed after this start location tweak?

Reply:
Yes. This start location corresponds to the one in the [param.yaml](https://github.com/aws-robotics/aws-robomaker-small-house-world/blob/ros2/param/waffle_pi.yaml#L4) file